charts/snapshot-validation-webhook/values.yaml

```diff
@@ -24,6 +24,28 @@ tls:
 imagePullSecrets: []
 podAnnotations: {}
 
+NetworkPolicy:
```

Comment:
Convention is that this should be spelled `networkPolicy`

Reply:
Agreed; I will correct this.